src/components/Buttons/ScrollToTopButton.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-fab-transition>
+    <v-btn
+      v-show="isScrolled"
+      color="primary"
+      fab
+      dark
+      bottom
+      right
+      fixed
+      @click="scrollToTop"
+    >
+      <v-icon>mdi-chevron-up</v-icon>
+    </v-btn>
+  </v-fab-transition>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapState } from 'vuex';
+
+export default Vue.extend({
+  computed: {
+    ...mapState('page', ['isScrolled'])
+  },
+  methods: {
+    scrollToTop(): void {
+      window.scrollTo(0, 0);
```

Comment:
```suggestion
      window.scrollTo({top: 0});
```
a tad nitpicky, but I find it clearer :)